### DISCUSSION TITLE
windows pc

### DISCUSSION THREAD

**Discussion by artdelphi** [hazzelnuts/ntfy-for-delphi#36] [Q&A] [answered]
how to sent notify to windows desktop?

**Comment by p-samuel** [comment]
Hello @artdelphi, this is a Delphi library. It's supposed you have Delphi installed on your machine so you can utilize this library in a Delphi project. Another point is that you'll need to place the OpenSSL DLLs (for windows) on the same directory where the binaries are located. Any trouble with what has been written in the README files?

**Reply by mitchellhu** [reply]
install Win64 OpenSSL v3.1.0 on my Win10 pc, but not found libeay32.dll and ssleay32.dll,
so.What version OpenSSL should I install for ntfy-for-delphi?
sorry for my bad English ability and bad Delphi skill.

**Reply by p-samuel** [reply]
Hi @mitchellhu, inside `sample` directory there is another folder called `openssl` where you can find the openssl binaries. You can also take a look on Indy's public repository to find the appropriate one. It's always a good choice to use the latest one.

https://github.com/IndySockets/OpenSSL-Binaries

**Reply by p-samuel** [reply]
NetHTTP branch is on development and in a few weeks I'll be releasing it. You no longer will need to use these libraries except if you stick to it.

**Reply by mitchellhu** [reply]
samuel ：
    Thank you for your quick reply. 
 very look for your new version.
To me, this project is a new world.
I'm trying to figure out all of it.

**Comment by p-samuel** [comment, marked as answer]
@artdelphi, if you want, you can watch the first session of this video where I explain briefly how to setup in your Delphi project. 

https://www.youtube.com/live/xzfiy01Q8o0?feature=share
